examples/online_serving/pooling/openai_pooling_client.py

```diff
@@ -23,7 +25,7 @@ def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument("--host", type=str, default="localhost")
     parser.add_argument("--port", type=int, default=8000)
-    parser.add_argument("--model", type=str, default="jason9693/Qwen2.5-1.5B-apeach")
+    parser.add_argument("--model", type=str, default="internlm/internlm2-1_8b-reward")
```

Comment:
I haven't checked but this example might be run in CI. If it is, we should keep using the smaller model